components/renku_data_services/notifications/api.spec.yaml

```
@@ -82,8 +82,140 @@ paths:
           $ref: "#/components/responses/Error"
       tags:
         - alerts
+  /alerts/webhook/alertmanager:
```

I think I would prefer that the path is not under `/alerts`: we can add support for any types of webhooks, what they trigger will depend on the internal logic.


```suggestion
  /webhooks/alertmanager:
```